src/Form/Input.js

```diff
@@ -112,6 +172,12 @@ class Input extends Component {
     onBlur() {},
     onChange() {},
     floating: false,
+    leftIconSize: 16,
+    leftIconColor: 'greyDarkest',
+    onLeftIconClick: null,
```

Comment:
Would it make sense to have empty functions as default values? 
```suggestion
    onLeftIconClick: () => {},
```

Reply:
I'm using the null value to check if there is a function or not for the icons. If so, the icon will have `cursor: pointer` added to the styles. Otherwise, the cursor will not change.